website/docs/sidebars.js

```diff
@@ -588,6 +588,22 @@ module.exports = {
           label: '実装',
           items: [
             'react-native/santoku/development/implement',
+            {
+              type: 'category',
+              label: 'スタイルガイド',
+              items: [
+                'react-native/santoku/development/implement/style-guide/overview',
+                'react-native/santoku/development/implement/style-guide/eslint',
+                'react-native/santoku/development/implement/style-guide/typescript',
+                'react-native/santoku/development/implement/style-guide/prettier',
+                'react-native/santoku/development/implement/style-guide/editorconfig',
+                'react-native/santoku/development/implement/style-guide/lint-ops',
+                'react-native/santoku/development/implement/style-guide/naming-conversion',
+                'react-native/santoku/development/implement/style-guide/typescript-style-guide',
```

Comment:
`TypeScriptコーディング規約` がメインで
`コーディングに関する命名規約` がサブっぽいので、順序を逆にするのはどうでしょう？


```suggestion
                'react-native/santoku/development/implement/style-guide/typescript-style-guide',
                'react-native/santoku/development/implement/style-guide/naming-conversion',
```

Reply:
↓で順序を逆にしてみましたー
0004ad0b7cce1c7276a3701a183239513e6e00bc